launcher/src/main.rs

```diff
@@ -342,9 +342,18 @@ struct Args {
 
     /// Whether you want to compile the model into a CUDA graph.
     /// This will speed up decoding but increase GPU memory usage.
-    #[clap(long, env, value_enum)]
+    /// Only use either `--compile` or `--eager`. Using both at the same time will
+    /// result in an error.
+    #[clap(default_value = "true", long, env, value_enum)]
```

Comment:
Since this defaults to true here, will this raise an error if the user specifies `--eager` without setting anything for `compile`?

Typically we would change the type of `compile` here to `Option[bool]` so we can distinguish between explicitly specified args vs unspecified.

Reply:
How do we detect the `--compile` is not provided?